.github/workflows/test.yaml

```diff
@@ -180,14 +180,38 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        feature: [alpine-3.16, alpine-3.17-openssl1.1-compat, alpine-latest]
+        feature:
+          - alpine-3.16-amd64-openssl-1.1.x
+          - alpine-3.17-amd64-openssl-3.0.x
+          - alpine-latest-amd64-openssl-3.0.x
+          - debian-bullseye-amd64-openssl-1.1.x
+          - debian-buster-amd64-openssl-1.1.x
+          - debian-latest-amd64-openssl-1.1.x
+          
+          # Add back once we solve https://github.com/prisma/prisma-private/issues/205
+          # - debian-latest-arm64-openssl-1.1.x
```

Comment:
We could also have it before we fix it? (it would fail, but we'll instantly know once it's fixed)

Reply:
We could, but I'd be wary of merging something that results in a red CI status